tasks/sign-index-image/README.md

```diff
@@ -28,6 +29,10 @@ data:
         configMapName: <configmap name>
 ```
 
+## Changes in 4.1.0
+* Updated task to truly support the `simple-signing-pipeline` request
```

Comment:
First of all, `simple-signing-pipeline` is also used for IRs now, so that's not a good way to describe it if you mean internal-pipelinerun. But also, I thought it was just missing something. But it seems this task didn't support it at all until this PR? It didn't even read `.sign.requestType`... So why the word `truly` and why `fix` in the commit title? It's more of a new feature, no? (Although this might have been an omission when implementing internal-pipelinerun.) 

And also, since this task just uses IRs right now, the params added in the RPA actually don't have any effect, so nothing will break, right? (It would be breaking stuff if a different pipeline was used, but both use simple-signing-pipeline now.)

Reply:
> First of all, `simple-signing-pipeline` is also used for IRs now, so that's not a good way to describe it if you mean internal-pipelinerun. 

Yes. I guess I mean `internal-pipelinerun` requestType. And the only request that is applicable for `internal-pipelinerun`s is the `simple-signing-pipeline`. But I'll re-word.

> But also, I thought it was just missing something. But it seems this task didn't support it at all until this PR? It didn't even read `.sign.requestType`... So why the word `truly` and why `fix` in the commit title? It's more of a new feature, no? (Although this might have been an omission when implementing internal-pipelinerun.)

I thought I made some changes to support internal-pipelinerun requestTypes but maybe I am wrong.
If you feel strongly about the work `fix`, I can switch to `feature`.

> And also, since this task just uses IRs right now, the params added in the RPA actually don't have any effect, so nothing will break, right? (It would be breaking stuff if a different pipeline was used, but both use simple-signing-pipeline now.)

the param in the RPA that does cause an issue is `request`. But I think https://github.com/hacbs-release/app-interface-deployments/pull/254 fixes that.

Reply:
Thanks for the changes.